.devcontainer/Dockerfile

```diff
@@ -0,0 +1,65 @@
+FROM debian:bullseye
+ENV DEBIAN_FRONTEND=noninteractive
+ENV LC_ALL=C.UTF-8
+ENV LANG=C.UTF-8
+RUN apt-get update \
+    && apt-get install -y vim nano git curl gcc ninja-build cmake libudev-dev \
+    python3 python3-pip libusb-1.0-0 libssl-dev pkg-config libtinfo5 clang \
+    && apt-get clean -y && rm -rf /var/lib/apt/lists/* /tmp/library-scripts
+RUN pip3 install websockets==10.2
+ARG CONTAINER_USER=gitpod
```

Comment:
Do you know if it's possible to specify the user for Gitpod via `.gitpod.yml` or is it always a user named `gitpod`? I checked the documentation but could not find anything in my initial search related to that.

One option I see is to specify the Docker user explicitly (for example `"esp"`), regardless of the environment. This way the user would be consistent for docker CLI, VS Code devcontainer, Gitpod, etc. Some parts of the code also require a specific user to be present, for example file `wowki/wowki-server.py` expects its paths to contain name `vscode`.